### PR TITLE
Change to 3 steps

### PR DESCRIPTION
**Pull request by loneil**
# Description
https://apps.nrs.gov.bc.ca/int/jira/browse/SHOWCASE-910
Reduce to 3 steps, add documentation in final step.

## Types of changes

New feature (non-breaking change which adds functionality)

## Checklist

- [x] I have read the [CONTRIBUTING](CONTRIBUTING.md) doc
- [x] I have checked that unit tests pass locally with my changes
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by bcgov-nr-csst** [comment]
SonarQube Analysis: https://sonarqube-k8vopl-tools.pathfinder.gov.bc.ca/dashboard?id=nr-get-token-pr-181

File: app/frontend/src/store/modules/apiAccess.js [hunk 1]
```diff
@@ -30,7 +30,16 @@ export default {
...
     generatedPassword: state => state.generatedPassword,
-    step: state => state.step
+    step: state => state.step,
+    tokenEndpoint(state) {
+      let prefix = 'sso';
+      if (state.environment.toLowerCase() === 'dev') {
+        prefix += '-dev';
+      } else if (state.environment.toLowerCase() === 'test') {
+        prefix += '-test';
+      }
```
Comment (jujaga) [inline app/frontend/src/store/modules/apiAccess.js:40]: Could do something like this to remove the need to do multiple conditionals - just leverage state.environment directly?
```js
if (['dev','test'].includes(state.environment.toLowerCase())) {
  prefix += `-${state.environment.toLowerCase()}`
}
```

File: app/frontend/src/components/apiAccess/ResourcesSidebar.vue [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+      <strong>Postman Collection</strong>
+      <v-btn color="primary" icon large>
+        <v-icon>cloud_download</v-icon>
+      </v-btn>
+    </a>
+    <hr class="mt-2" />
+    <h3 class="mb-2">Note</h3>
+    <p>
+      This service client can be used to make REST API calls and an example Postman collection is provided.
+      Fetch a new toekn using the service client and password against the authorization endpoint (see postman collection)
```
Comment (TimCsaky) [inline app/frontend/src/components/apiAccess/ResourcesSidebar.vue:34]: typo! ( > 'token')

Comment (loneil) [reply app/frontend/src/components/apiAccess/ResourcesSidebar.vue]: 👍

**Comment by bcgov-nr-csst** [comment]
SonarQube Analysis: https://sonarqube-k8vopl-tools.pathfinder.gov.bc.ca/dashboard?id=nr-get-token-pr-181

File: app/frontend/src/store/modules/apiAccess.js [hunk 3]
```diff
@@ -30,7 +30,14 @@ export default {
...
     generatedPassword: state => state.generatedPassword,
-    step: state => state.step
+    step: state => state.step,
+    tokenEndpoint(state) {
+      let prefix = 'sso';
+      if (['dev', 'test'].includes(state.environment.toLowerCase())) {
+        prefix += `-${state.environment.toLowerCase()}`;
+      }
+      return `https://${prefix}.pathfinder.gov.bc.ca/auth/realms/jbd6rnxw/protocol/openid-connect/token`;
+    }
```
Comment (jujaga) [inline app/frontend/src/store/modules/apiAccess.js:40]: Now that I look at this again, I now realize this is just trying to generate the general openid-connect token endpoint. We already have the base auth url defined at `config.keycloak.serverUrl` (accessible via $config.keycloak.serverUrl) which would return for example `https://sso-dev.pathfinder.gov.bc.ca/auth` and change based on environment already. Assuming I didn't botch how to access that variable, it'd look something like this...

```js
tokenEndpoint(state) {
  return `${this._vm.$config.keycloak.serverUrl}/realms/jbd6rnxw/protocol/openid-connect/token`;
}

**Pull request merged**